### ISSUE TITLE
Argo tunnel not working 

### ISSUE THREAD

**Issue by nekosumi** [cloudflare/cloudflared#339]
config.yaml

credentials-file: /home/****/.cloudflared/*****.json
tunnel: *****
hostname: ****.xyz
url: http://127.0.0.1:8080
logfile: /var/log/cloudflared.log

Logs:

[*(&^#@localhost ~] cloudflared tunnel run
2021-03-23T12:24:28Z INF Starting tunnel tunnelID=****
2021-03-23T12:24:28Z INF Version 2021.3.1
2021-03-23T12:24:28Z INF GOOS: linux, GOVersion: devel +11087322f8 Fri Nov 13 03:04:52 2020 +0100, GoArch: amd64
2021-03-23T12:24:28Z INF Settings: map[cred-file:/home/====/.cloudflared/====.json credentials-file:/home/====/.cloudflared/====.json hostname:====.xyz logfile:/var/log/cloudflared-====.log url:http://127.0.0.1:8080]
2021-03-23T12:24:28Z INF cloudflared will not automatically update when run from the shell. To enable auto-updates, run cloudflared as a service: https://developers.cloudflare.com/argo-tunnel/reference/service/
2021-03-23T12:24:28Z INF Generated Client ID: ****
2021-03-23T12:24:28Z INF Initial protocol h2mux
2021-03-23T12:24:28Z INF Starting metrics server on 127.0.0.1:42987/metrics
2021-03-23T12:24:29Z INF Connection **** registered connIndex=0 location=WAW
2021-03-23T12:24:30Z INF Connection **** registered connIndex=1 location=RIX
2021-03-23T12:24:31Z INF Connection **** registered connIndex=2 location=WAW
2021-03-23T12:24:32Z INF Connection **** registered connIndex=3 location=RIX

Error 1033
Argo Tunnel error

What is wrong?

**Comment by nmldiegues** [comment]
Hello @nekosumi ,

You are using Named Tunnels (which is the recommended way). You performed steps [1 and 3 but forgot step 2](https://developers.cloudflare.com/cloudflare-one/connections/connect-apps) to [Route to the Tunnel](https://developers.cloudflare.com/cloudflare-one/connections/connect-apps/routing-to-tunnel/dns).

I see you have`hostname: ...` in the configuration, but that is not used in Named Tunnels. It is from the old tunnels that were not provisioned a priori (and are thus not recommended for [various reasons](https://blog.cloudflare.com/argo-tunnels-that-live-forever/);  not to mention that [new features](https://blog.cloudflare.com/many-services-one-cloudflared/) are only developed and released for Named Tunnels).

TLDR:
 - remove `hostname: ...` as it is doing nothing
 - run `cloudflared tunnel route ip --help` and follow that --- check out the links above for the documentation

**Comment by nekosumi** [comment]
I have no idea what to do

**Comment by nekosumi** [comment]
> Hello @nekosumi ,
> 
> You are using Named Tunnels (which is the recommended way). You performed steps [1 and 3 but forgot step 2](https://developers.cloudflare.com/cloudflare-one/connections/connect-apps) to [Route to the Tunnel](https://developers.cloudflare.com/cloudflare-one/connections/connect-apps/routing-to-tunnel/dns).
> 
> I see you have`hostname: ...` in the configuration, but that is not used in Named Tunnels. It is from the old tunnels that were not provisioned a priori (and are thus not recommended for [various reasons](https://blog.cloudflare.com/argo-tunnels-that-live-forever/); not to mention that [new features](https://blog.cloudflare.com/many-services-one-cloudflared/) are only developed and released for Named Tunnels).
> 
> TLDR:
> 
> * remove `hostname: ...` as it is doing nothing
> * run `cloudflared tunnel route ip --help` and follow that --- check out the links above for the documentation
I have no idea what to do

**Comment by nmldiegues** [comment]
You seem to have double posted to https://community.cloudflare.com/t/argo-tunnel-not-working/254605/2

I have re-iterated on the answer. Have you read the documentation links that I provided? You seem to be missing Step 2, to "Route traffic to your Tunnel", from https://developers.cloudflare.com/cloudflare-one/connections/connect-apps

**Comment by nmldiegues** [comment]
Apologies:  I previously had written a command wrong above.  I have edited the post.

**Comment by Anondmg** [comment]
I need help fixing mine, when I click my links it show a white Argo tunnel error page 1033

**Comment by sefinek24** [comment]
Show your config file

**Comment by abelinkinbio** [comment]
Generally, a `1033 error` indicates the Tunnel is not connected to Cloudflare. Would you visiting the Zero Trust dashboard or running `cloudflared tunnel list` to cofirm that the Tunnel is running and listed as active. If not, could you try the troubleshooting steps outlined here: https://developers.cloudflare.com/cloudflare-one/faq/teams-troubleshooting/#i-see-an-error-1033-when-attempting-to-run-a-tunnel
